src/ATen/native/xpu/sycl/BatchNormKernels.cpp

```diff
@@ -1008,9 +1008,9 @@ struct BatchNormCollectStatisticsChannelsLastKernelFunctor
   }
 
   BatchNormCollectStatisticsChannelsLastKernelFunctor(
-      const scalar_t* __restrict__ input,
-      accscalar_t* __restrict__ out_mean,
-      accscalar_t* __restrict__ out_invstd,
+      const scalar_t* input,
+      accscalar_t*  out_mean,
+      accscalar_t*  out_invstd,
```

Comment:
The changes impact Linux path. You can declare a macro to use here, and distinguish Win/Linux in the macro.

Reply:
Refer to here https://github.com/intel/torch-xpu-ops/blob/1d70431c072db889d9a47ea4956049fe340a426d/src/ATen/native/xpu/sycl/TensorTransformationsKernels.cpp#L9-L13